shell/common/shell.cc

```diff
@@ -501,7 +503,8 @@ std::unique_ptr<Shell> Shell::Spawn(
           return engine->Spawn(/*delegate=*/delegate,
                                /*dispatcher_maker=*/dispatcher_maker,
                                /*settings=*/settings,
-                               /*animator=*/std::move(animator));
+                               /*animator=*/std::move(animator),
+                               /*initial_route*/ initial_route);
```

Comment:
s/initial_route/initial_route= 

I wish the linter worked for that, doesn't seem to be.

Reply:
Done